deployments/helm-chart/values.yaml

```diff
@@ -74,6 +74,17 @@ controller:
   ## The termination grace period of the Ingress controller pod.
   terminationGracePeriodSeconds: 30
 
+  ## The resources of the Ingress controller pods.
+  resources: {}
+  # To specify resources, uncomment the following lines, adjust them as necessary,
+  # and remove the curly braces after 'resources:'.
+  #  limits:
```

Comment:
This indentation level is a bit inconsistent with what we already have in this file.
I'd prefer something like:
```yaml
  resources: {}
    # limits:
    #   cpu: 100m
    #   memory: 64Mi
    # requests:
    #   cpu: 100m
    #   memory: 64Mi
```

Reply:
Done! I think the indentation is right now :)